src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('span.population');
+
+const populations = Array.from(populationElements).map(
+  // eslint-disable-next-line
+    (el) => +el.textContent.replace(/,/g, '')
+);
+
+const totalPopulation = populations.reduce((acc, num) => acc + num, 0);
+
+const averagePopulation = totalPopulation / populations.length;
+
+const totalPopSpan = document.querySelector('span.total-population');
+const averagePopSpan = document.querySelector('span.average-population');
```

Comment:
Ensure that there is an element with the class 'average-population' in the DOM. If this element does not exist, this line will result in a null reference error when trying to set the textContent.